rollup/ccc/async_checker_test.go

```diff
@@ -26,6 +26,8 @@ func TestAsyncChecker(t *testing.T) {
 	// Create a database pre-initialize with a genesis block
 	db := rawdb.NewMemoryDatabase()
 	chainConfig := params.TestChainConfig.Clone()
+	chainConfig.EuclidTime = nil
+	chainConfig.EuclidV2Time = nil
```

Comment:
This test will be broken if TestChainConfig has `EuclidTime` and `EuclidV2Time` added as `0`s.